manifests/t/TianZhen/NeatChat/1.1.17/TianZhen.NeatChat.installer.yaml

```
@@ -0,0 +1,27 @@
+# Created with YamlCreate.ps1 Dumplings Mod
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: TianZhen.NeatChat
+PackageVersion: 1.1.17
+UpgradeBehavior: install
+ReleaseDate: 2025-03-10
+Installers:
+- Architecture: x64
+  InstallerType: nullsoft
+  Scope: user
+  InstallerUrl: https://github.com/tianzhentech/NeatChat/releases/download/v1.1.17/NeatChat_1.1.17_x64-setup.exe
+  InstallerSha256: EE4FC74DF0939359E233A05569560FF1AD42CF4A7D311F7EAB11B5D337CC2588
+  ProductCode: NeatChat
+- Architecture: x64
+  InstallerType: wix
+  Scope: machine
+  InstallerUrl: https://github.com/tianzhentech/NeatChat/releases/download/v1.1.17/NeatChat_1.1.17_x64_en-US.msi
+  InstallerSha256: 677D32696E7BAE1C46160F2A828B5FF38F0A425CBC79FA302F275A2346EA96B1

```

```suggestion
  InstallerSha256: 6A629A37F5B8555F3D2A909EC7E084CDDA205D5FCAD0FFE3A35FB0377F033FD4
```

(Automated response - build 998.)